RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift

```diff
@@ -312,6 +300,19 @@ private extension View {
 		)
 	}
 
+	private func unknownComponents(with destinationStore: PresentationStoreOf<TransactionReview.Destination>) -> some View {
+		sheet(
+			store: destinationStore,
+			state: /TransactionReview.Destination.State.unknownDappComponents,
+			action: TransactionReview.Destination.Action.unknownDappComponents,
+			content: { store in
+				NavigationStack {
```

Comment:
You can use the .inNavigationStack modifier, then you don't need `store in`. Also, shouldn't it have a medium detent?